app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(
+        list_of_players: list[Player]
+) -> int:
+    if list_of_players:
+        return sum(
+            [player.get_rating() for player in list_of_players]
```

Comment:
you can use a generator expression instead of a list comprehension. simply remove [ ]